### PR TITLE
Remove OS Update column

### PR DESCRIPTION
**Pull request by nicolethoen**
[CONSOLE-1185](https://jira.coreos.com/browse/CONSOLE-1185)

screen shot 2019-01-17 at 10 29 28 am
screen shot 2019-01-17 at 10 29 46 am
screen shot 2019-01-17 at 10 30 00 am

### CODE REVIEW

**Comment by jhadvig** [comment]
Just for info, why are we removing the column ?

**Comment by spadgett** [comment]
> Just for info, why are we removing the column ?

The value it currently shows is Tectonic-specific and never set on OpenShift clusters.

File: frontend/public/components/node.tsx [hunk 1]
```diff
@@ -78,23 +77,14 @@ const NodeCLUpdateStatus = ({node}) => {
   </div>;
 };
 
-const NodeCLStatusRow = ({node}) => {
-  const updateStatus = containerLinuxUpdateOperator.getUpdateStatus(node);
-  return updateStatus ? <span>{updateStatus.className && <span><i className={updateStatus.className}></i>&nbsp;&nbsp;</span>}{updateStatus.text}</span> : null;
-};
-
 const NodeRow = ({obj: node, expand}) => {
-  const isOperatorInstalled = containerLinuxUpdateOperator.isOperatorInstalled(node);
```
Comment (spadgett) [inline frontend/public/components/node.tsx]: We should remove the corresponding `containerLinuxUpdateOperator` code if it's dead code.

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**